obspy/io/mseed/tests/test_mseed_util.py

```diff
@@ -648,6 +648,26 @@ def test_time_shifting(self, testdata):
             st_before[0].stats.starttime -= 33.3
             assert st_before == st_after
 
+            # Test spread_time_over_file().
```

Comment:
In general it's better to have tests for separate functionality also in separate test cases, so please put a test method around this part.